### DISCUSSION TITLE
vendir: Fetch helm-chart from OCI repository with authentication

### DISCUSSION THREAD

**Discussion by thriqon** [renovatebot/renovate#31458] [Request Help]
### What would you like help with?

Other

### How are you running Renovate?

Self-hosted

### If you're self-hosting Renovate, tell us which platform (GitHub, GitLab, etc) and which version of Renovate.

v38.87.1 / Bitbucket Server

### Please tell us more about your question or problem

I'm using vendir to update helm-charts from various repositories without authentication. For a new item I have to use authentication, though, and I'm wondering if this is supported at all.

Vendir by itself supports authentication by referencing `Secret`s in the same or a different file, but I found no way to configure this with renovate.

### Logs (if relevant)

<details><summary>Logs</summary>

```

Replace this text with your logs, between the starting and ending triple backticks

```

</details>

**Comment by secustor** [comment]
https://docs.renovatebot.com/getting-started/private-packages/
